src/pybind/mgr/cephadm/tests/fixtures.py

```diff
@@ -87,30 +87,8 @@ def osd_obj():
 
 
 def wait(m, c):
-    # type: (CephadmOrchestrator, Completion) -> Any
-    m.process([c])
-
-    try:
-        import pydevd  # if in debugger
-        in_debug = True
-    except ImportError:
-        in_debug = False
-
-    if in_debug:
-        while True:    # don't timeout
-            if c.is_finished:
-                raise_if_exception(c)
-                return c.result
-            time.sleep(0.1)
-    else:
-        for i in range(30):
-            if i % 10 == 0:
-                m.process([c])
-            if c.is_finished:
-                raise_if_exception(c)
-                return c.result
-            time.sleep(0.1)
-    assert False, "timeout" + str(c._state)
+    # type: (CephadmOrchestrator, OrchResult) -> Any
+    return raise_if_exception(c)
```

Comment:
Does this method still need to exist?  (Or should it be renamed?)

Reply:
Right. it's still needed due to the sub-interpreter magic: 

https://github.com/ceph/ceph/blob/3b42756c1b1846fa873af50a1b34f838428e2330/src/pybind/mgr/orchestrator/_interface.py#L215-L226